.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
```

Comment:
could you add a section for `github-actions` to update the workflows external actions too